### PR TITLE
Update node documentation - multichain

### PR DESCRIPTION
**Pull request by HildisviniOttar**
Updated instructions for node operators. Includes:
- Dealing with slash
- Fixing unstuck transactions
- Manual LEAVE
- Yggdrasil administration
- Community Links
- `make reset` command

### CODE REVIEW

File: thornodes/alerting.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ description: Telegram Notification Service
 
 # Alerting
 
+## Community alert channels
+
+To listen for update announcements, join the following channels: 
+* Telegram **THORNode Announcements** [https://t.me/thornode_ann](https://t.me/thornode_ann)
+* Discord **THORChain Community Devs** [https://discord.gg/kvZhpEtHAw](https://discord.gg/kvZhpEtHAw)
```
Comment (veado) [inline thornodes/alerting.md:11]: [nitpick] For Discord: especially `#thornode-mccn` /` #thornode-mctn` channels

File: thornodes/leaving.md [hunk 2]
```diff
@@ -50,29 +58,84 @@ This ensures you can safely leave this system if you no longer have access to yo
...
+
+{% hint style="warning" %}
+**Do not cache** inbound_addresses. These are only valid for a short period of time. Always refresh to get the latest before sending funds.  
+{% endhint %}
+
+{% hint style="warning" %}
+Use the **address** field. Chains with a **router** present such as ETH need to send funds via the router smart-contract. Paste the router address into etherscan, click "Contract" and "Write Contract" and use a Web3 wallet to connect. 
+{% endhint %}
+
+The **memo** required is `YGGDRASIL-:<BlockHeight>`. For example `YGGDRASIL-:782412`. The block height can be found from the `status_since` field here:
```
Comment (veado) [inline thornodes/leaving.md:120]: If I'm not wrong, `memo` has to be hex encoded (at least by using MetaMask - [web3.toHex](web3.toHex) helps here, you can encode the memo by using `web3.toHex` online https://www.mobilefish.com/download/ethereum/web3api.html)

File: thornodes/managing.md [hunk 3]
```diff
@@ -289,21 +297,179 @@ It is up to the node operator to setup those extra backups of the core volumes t
...
+CANCELLING 30 for 0x3eb68bF15A7A6769219A66C5c493fa7C40511E19
+0xccdd1b9d8e6ac0c7314fa39f78442c7702688db56091e8d379da75a3cfbce936
+CANCELLING 29 for 0x3eb68bF15A7A6769219A66C5c493fa7C40511E19
+0x43bad098782f7bac68e401390ef300dc97d9d1d1b322eb566de1ff06b2cf9b21
+```
+14. `make restart` and choose `ethereum-daemon`
+
+#### Constantly accumulating slash points
+
+**Problem:** Sometimes bifrost fails to forward observations to thornode,due to an account number / sequence number mismatch. Here is what you need to check:
```
Comment (veado) [inline thornodes/managing.md:471]: (Issue https://gitlab.com/thorchain/thornode/-/issues/950)

File: thornodes/alerting.md (hunk 1 above)
Comment (HildisviniOttar) [reply thornodes/alerting.md]: Updated

**Pull request merged**